index.html

```diff
@@ -1962,8 +1962,20 @@ <h3>Verify Proof</h3>
 integrity of a <a>secured data document</a> by verifying its
 digital proof. Required inputs are a
 <a>secured data document</a> (<var>securedDocument</var>) and
-<a>proof options</a> (<var>options</var>). A <dfn>verification result</dfn> is
-produced as output.
+<a>proof options</a> (<var>options</var>). The <a>proof options</a> MAY contain
+the following values that help protect against relay and replay attacks:
+<var>expectedProofPurpose</var>,
+<var>acceptableCreatedTimeDeviationInSeconds</var>, <var>domain</var>, and
+<var>challenge</var>. The <var>expectedProofPurpose</var> value is used to
+ensure that the <var>proof</var> was generated by the sender for the expected
+reason by the receiver, such as "authentication". The
```

Comment:
I don't think `sender` is quite right terminology. If we just look at the usage of DI with VCs for example, the issuer will create a proof that they send to the holder, but then the holder becomes the sender when presenting it to a verifier. The holder would not get to change the proof purpose for the proof created by the issuer in that case. Perhaps "proof creator" would be better here -- and `receiver` should then be `verifier`.

```suggestion
ensure that the <var>proof</var> was generated by the proof creator for the expected
reason by the verifier, such as "authentication". The
```